views/lobby.ejs

```diff
@@ -192,6 +192,10 @@
 								<br>
 							
 								<button id="playerHighlightColourButton">Click to change highlight colour</button>
+								<br>
+								<button id="setHighlightColorToYellowButton" onclick="setAllHighlightColorsToYellow()">Set all highlight colors to yellow</button>
```

Comment:
Perhaps have this as a checkbox option in the display options? Instead of a button?